scripts/save_org_product_data_daily.pl

```diff
@@ -0,0 +1,126 @@
+
+ensure_dir_created($BASE_DIRS{ORGS});
+
+my $products_collection = get_products_collection();
+my $orgs_collection = get_orgs_collection();
+
+sub get_org_data ($org_id) {
+	my $org_data = $products_collection->aggregate(
+		[
+			{'$match' => {'owner' => $org_id}},
+			{
+				'$group' => {
+					'_id' => '$owner',
+					'number_of_products' => {'$sum' => 1},
+					'number_of_data_quality_errors' => {
+						'$sum' => {'$size' => '$data_quality_errors_tags'}
+					},
+					'number_of_data_quality_warnings' => {
+						'$sum' => {'$size' => '$data_quality_warnings_tags'}
+					},
+					'number_of_products_without_nutriscore' => {
+						'$sum' => {
+							'$cond' => [{'$in' => ['en:nutriscore-not-computed', '$misc_tags']}, 1, 0]
+						}
+					},
+					'opportunities_to_improve_nutriscore' => {
+						'$sum' => {
+							'$cond' => [{'$in' => ['possible-improvements', '$misc_tags']}, 1, 0]
+						}
+					},
+					'products_with_changes_since_last_export' => {
+						'$sum' => {
+							'$cond' => [{'$regexMatch' => {input => '$states', regex => 'en:to-be-exported'}}, 1, 0]
+						}
+					},
+					'date_of_last_update' => {'$max' => '$last_modified_t'}
+				}
+			}
+		]
+	)->next;
+
+	my $number_of_products = $org_data->{number_of_products} // 0;
+	my $number_of_products_without_nutriscore = $org_data->{number_of_products_without_nutriscore} // 0;
+	my $number_of_products_with_nutriscore = $number_of_products - $number_of_products_without_nutriscore;
+	my $percentage_of_products_with_nutriscore
+		= $number_of_products > 0 ? ($number_of_products_with_nutriscore / $number_of_products) * 100 : 0;
+
+	return {
+		'products' => {
+			'number_of_products' => $number_of_products,
+			'number_of_data_quality_errors' => $org_data->{number_of_data_quality_errors} // 0,
+			'number_of_data_quality_warnings' => $org_data->{number_of_data_quality_warnings} // 0,
+			'number_of_products_without_nutriscore' => $number_of_products_without_nutriscore,
+			'percentage_of_products_with_nutriscore' => $percentage_of_products_with_nutriscore,
+			'opportunities_to_improve_nutriscore' => $org_data->{opportunities_to_improve_nutriscore} // 0,
+			'products_with_changes_since_last_export' => $org_data->{products_with_changes_since_last_export} // 0,
+			'date_of_last_update' => $org_data->{date_of_last_update} // 0,
+		},
+	};
+}
+
+sub update_org_data ($org_id) {
+
+	my $data = get_org_data($org_id);
+
+	$orgs_collection->update_one({'org_id' => $org_id}, {'$set' => {'data' => $data}}, {'upsert' => 1});
+
+	my $org_file_path = "$BASE_DIRS{ORGS}/$org_id.sto";
+	my $org_ref = retrieve_org($org_id);
+
+	$org_ref->{'data'} = $data;
+
+	store($org_ref, $org_file_path);
+}
+
+sub gather_org_data {
+	my @orgs = $orgs_collection->find()->all();
+	my $count = scalar @orgs;
+	my $i = 0;
+
+	foreach my $org (@orgs) {
+		my $org_id = $org->{'org_id'};
+		print "Processing organization $i/$count: $org_id\n";
+		update_org_data($org_id);
```

Comment:
Looks good, but you did not apply @alexgarel 's suggestion for the eval:

https://github.com/openfoodfacts/openfoodfacts-server/pull/10447#discussion_r1674182902